apps/davi/src/hooks/Guilds/useProposalMetadata.ts

```diff
@@ -1,12 +1,27 @@
 import useIPFSFile from 'hooks/Guilds/ipfs/useIPFSFile';
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import { ProposalMetadata } from 'types/types.guilds';
 import { useOrbisContext } from 'contexts/Guilds/orbis';
+import * as contentHashLib from '@ensdomains/content-hash';
 
 function useProposalMetadata(contentHash: string) {
   const { orbis } = useOrbisContext();
   const [orbisData, setOrbisData] = useState<any>();
 
+  // Support backwards compatible encoded ipfs hashes
+  const { decodedContentHash } = useMemo(() => {
+    if (!contentHash || contentHash?.includes('://')) console.log('hey');
+
+    try {
+      return {
+        decodedContentHash: contentHashLib.decode(contentHash),
+      };
+    } catch (e) {
+      console.error(e);
+      return { decodeError: e };
+    }

```

Comment:
We aren't doing anything with the `decodeError` information. Should we remove the return or use it?